src/helper/task.ts

```diff
@@ -215,55 +215,47 @@ export class ZkWasmServiceHelper {
   }
 
   async loadTasks(query: QueryParams): Promise<PaginationResult<Task[]>> {
-    let headers = { "Content-Type": "application/json" };
-    let queryJson = JSON.parse("{}");
-
-    // Set default total to 2 if not provided
-    const defaultQuery = {
-      total: 2,
-    };
-
-    // Merge the original query with default values
-    const mergedQuery = { ...defaultQuery, ...query };
-
     // Validate query params
-    if (mergedQuery.start != null && mergedQuery.start < 0) {
+    if (query.start != null && query.start < 0) {
```

Comment:
please still keep the orignal 
 Set default total to 2 if not provided
    const defaultQuery = {
      total: 2,
    };
    
logic